omod/src/main/java/org/openmrs/module/registrationapp/fragment/controller/field/DropDownFragmentController.java

```diff
@@ -25,8 +25,7 @@ public void controller(FragmentConfiguration config) throws Exception {
             Concept conceptSet = Context.getConceptService().getConceptByUuid(conceptSetUuid);
 
             for (Concept concept: conceptSet.getSetMembers()) {
-            	Map < String, Object > option = new HashMap < String, Object > ();
-                option = new HashMap < String, Object > ();
+            	Map < String, Object > option = new HashMap <String, Object > ();
```

Comment:
```suggestion
                Map < String, Object > option = new HashMap<>();
```
Since java version 7, there is a diamond operator, doesn't the code look clearer when we use it?

Additionally, please note that this particular file uses spaces, so when you use a tab, the code becomes inconsistent at the file level.

Reply:
@sacull, I made the requested changes and the build fails since Diamond operators are not supported in source - 1.6, 

which has been set in the Travis build settings. 
So, I'm thinking of reverting the changes.